### PR TITLE
Add absoluteMaxShort to hyperdrive-wasm bindings

### LINKED ISSUE #1259: Add `calculate_absolute_max_short` to hyperwasm and refactor ReadHyperdrive.getMaxShort() to use it

### PR DESCRIPTION
**Pull request by DannyDelott**
Closes #1259

### CODE REVIEW

File: crates/hyperdrive-wasm/src/short/max.rs [hunk 1]
```diff
@@ -46,3 +46,29 @@ pub fn maxShort(params: IMaxShortParams) -> Result<BigInt, Error> {
 
     result_fp.to_bigint()
 }
+
+///////////////////////////////////
```
Comment (ryangoree) [inline crates/hyperdrive-wasm/src/short/max.rs:50]: Left over comment

File: packages/hyperdrive-js-core/src/hyperwasm.ts [hunk 2]
```diff
@@ -3,7 +3,8 @@
 // invalidated and the latest version of the wasm package will be loaded.
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-import * as hyperwasm from "@delvtech/hyperdrive-wasm?v0.14.2";
+// import * as hyperwasm from "@delvtech/hyperdrive-wasm?v0.14.2";
+import * as hyperwasm from "@delvtech/hyperdrive-wasm";
```
Comment (ryangoree) [inline packages/hyperdrive-js-core/src/hyperwasm.ts:7]: Left over

**Review by ryangoree — APPROVED** [review]
🔥

File: crates/hyperdrive-wasm/src/short/max.rs [hunk 3]
```diff
@@ -46,3 +46,29 @@ pub fn maxShort(params: IMaxShortParams) -> Result<BigInt, Error> {
...
+#[ts(extends = IStateParams)]
+struct AbsoluteMaxShortParams {
+    /// The exposure of the pool's current checkpoint.
+    checkpoint_exposure: BigInt,
+    /// The maximum number of iterations to run the binary search for.
```
Comment (dpaiton) [inline crates/hyperdrive-wasm/src/short/max.rs:55]: it's not a binary search fwiw; it's "newtons method"

**Pull request merged**